Changes in module 'nf-core/trinity'
--- modules/nf-core/trinity/main.nf
+++ modules/nf-core/trinity/main.nf
@@ -9,7 +9,7 @@
         'biocontainers/trinity:2.15.1--pl5321h146fbdb_3' }"
 
     input:
-    tuple val(meta), path(reads, stageAs: "input*/*", arity: '1..*')
+    tuple val(meta), path(reads, stageAs: "input*/*")
 
     output:
     tuple val(meta), path("*.fa.gz")    , emit: transcript_fasta

************************************************************
