Changes in module 'nf-core/sortmerna'
--- modules/nf-core/sortmerna/main.nf
+++ modules/nf-core/sortmerna/main.nf
@@ -52,10 +52,10 @@
     $mv_cmd
     mv rRNA_reads.log ${prefix}.sortmerna.log
 
-    cat <<-END_VERSIONS > versions.yml
-    "${task.process}":
-        sortmerna: \$(echo \$(sortmerna --version 2>&1) | sed 's/^.*SortMeRNA version //; s/ Build Date.*\$//')
-    END_VERSIONS
+cat <<-END_VERSIONS > versions.yml
+"${task.process}":
+    sortmerna: \$(echo \$(sortmerna --version 2>&1) | sed 's/^.*SortMeRNA version //; s/ Build Date.*\$//')
+END_VERSIONS
     """
 
     stub:

************************************************************
